test/trello-provider.test.ts

```diff
@@ -0,0 +1,146 @@
+/* Copyright © 2021 Seneca Project Contributors, MIT License. */
+
+import * as Fs from 'fs'
+
+import TrelloProvider from '../src/trello-provider'
+
+
+const Seneca = require('seneca')
+const SenecaMsgTest = require('seneca-msg-test')
+const TrelloProviderMessages = require('./trello-provider.messages').default
+
+const CONFIG: any = {}
+
+if (Fs.existsSync(__dirname + '/local-config.js')) {
+    Object.assign(CONFIG, require(__dirname + '/local-config.js'))
```

Comment:
We also want the unit tests to pass in Github Actions - they currently fail because of course there is no API key. This is a problem we will solve later. For now, we just want the tests to "pass". Can you add a flat variable here to "turn off" any tests that fail if there is no key

Reply:
Indeed @rjrodger. Thank you for the suggestion. 

Therefore, I created a variable named "missingKeys", which initially has a value equal to true. Changed to false, only if the local-config.js file exists. 

And I added a conditional, checking the variable value in each test suite. If the missingKeys variable is equal to true we "skip" the test. And as you suggested, this conditional was applied only in the test suites that the credentials are necessary. 

Take a look again, please.